src/components/Login/Login.scss

```
@@ -0,0 +1,203 @@
+  position: relative;
+  text-align: center;
+  margin: auto 0;
+  max-width: 100vw;
+  padding: 16px;
+  width: 100%;
+
+  .login_footer {
+    display: flex;
+    flex-direction: column;
+
+    .login_footer_span {
+      font-size: 12px;
+      opacity: 0.5;
+      margin-bottom: 25px;
+    }
+
+    .login_footer_link {
+      font-size: 18px;
+      margin-bottom: 20px;
+
+      a {
+        color: $buttonColorBlack;
+        text-decoration: underline;
+        margin-left: 5px;
+      }
+    }
+  }
+  .login_form {
+    @include column();
+    text-align: left;
+    width: 360px;
+    margin: 0 auto;
+    color: #adacac;
+    font-size: 18px;
+
+    .email_box {
+      @include column();
+      margin-bottom: 25px;
+
+      .email_warning {
+        @include warning();
+        margin-top: 5px;
+      }
+
+      .email_span {
+        margin-bottom: 5px;
+      }
+    }
+    .password_box {
+      @include column();
+      margin-bottom: 20px;
+
+      .password_span {
+        margin-bottom: 5px;
+      }
+
+      .password_warning {
+        @include warning();
+        margin-top: 5px;
+      }
+    }
+  }
+  .login_form input {
+    height: 40px;
+    border: 1px solid #9f9f9f;
+    border-radius: 3px;
+    font-size: 16px;
+  }
+
+  .login_form::placeholder {
+    color: #adacac;
+    margin-left: 5px;
+  }
+  .login_main {
+    width: 500px;
+    height: 640px;
+    @include column();
+    margin: auto;
+    border: none;
+    position: relative;
+    z-index: 11;
+    background: #fff;
+
+    .login_continue_btn {
+      width: 360px;
+      height: 46px;
+      color: white;
+      border: none;
+      background-color: $buttonColorBlack;
+      font-weight: 700;
+      font-size: 16px;
+      margin-bottom: 20px;
+
+      &:hover {
+        background-color: $buttonColorYellow;
+        transition: ease-in-out 0.5s;
+        color: $buttonColorBlack;
+        background: linear-gradient(
+          to left,
+          $buttonColorYellow 50%,
+          $buttonColorBlack 50%
+        );
+        background-size: 200% 100%;
+        background-position: right bottom;
+        cursor: pointer;
+      }
+    }
+
+    .divide_line {
+      display: flex;
+      justify-content: center;
+      border-top: 1px solid #d9d5d2;
+      color: #d9d5d2;
+      height: 25px;
+      width: 360px;
+      margin: 0 auto;
+    }
+
+    .login_title {
+      position: relative;
+      font-size: 40px;
+      font-weight: 700;
+      padding: 24px 55px 15px 55px;
+
+      .title_btn {
+        position: absolute;
+        top: 10px;
+        right: 20px;
+        font-size: 25px;
+        border: none;
+        background: white;
+      }
+    }
+    .login_social button {
+      cursor: pointer;
+      height: 45px;
+      font-size: 16px;
+    }
+    .login_social {
+      display: flex;
+      flex-direction: column;
+      margin: 0 68px;
+
+      .login_social_span {
+        font-size: 18px;
+        margin-bottom: 25px;
+      }
```

```suggestion
      &_span {
        font-size: 18px;
        margin-bottom: 25px;
      }
```
부모선택자를 쓰면, 좀더 짧게 쓸 수 있습니다.
[Adding Suffixes](https://sass-lang.com/documentation/style-rules/parent-selector#adding-suffixes) 참고 해 주세요!